slick.dataview.js

```diff
@@ -542,36 +556,40 @@
       refresh();
     }
 
-    /**
-     * @param varArgs Either a Slick.Group's "groupingKey" property, or a
-     *     variable argument list of grouping values denoting a unique path to the row.  For
-     *     example, calling collapseGroup('high', '10%') will collapse the '10%' subgroup of
-     *     the 'high' group.
-     */
-    function collapseGroup(varArgs) {
```

Comment:
why did you remove the comments and the `varArgs`? If you thought this is not used at all, it's not completely true, it's used below with the `arguments` but doesn't call the `varArgs` directly, however the comments is still valid. So to remove confusion, the create added the `varArgs` for users who call this method, so I think it should all go back to it's original state, which is comments + arg.

Reply:
If you wish to put the comments back in I can. The removal of the parameter was because it wasn't being used. So really I was just tidying code with those removals.